SPECS/tdnf/tdnf.spec

```diff
@@ -121,23 +129,30 @@ systemd services for periodic automatic update
 %autosetup -p1
 
 %build
+%if 0%{?with_check}
+# remove problematic test file when running tests from within the rpm build directory
+rm pytests/tests/test_srpms.py
+%endif
+
+mkdir -p build && cd build
 %cmake \
     -DCMAKE_BUILD_TYPE=Debug \
     -DBUILD_SHARED_LIBS=OFF \
     -DCMAKE_INSTALL_LIBDIR:PATH=%{_libdir} \
-    -DHISTORY_DB_DIR=%{_tdnf_history_db_dir}
+    -DHISTORY_DB_DIR=%{_tdnf_history_db_dir} \
+	..
 
-%cmake_build
-
-cd %{__cmake_builddir}
+%make_build
 %make_build python
 
 %check
-pip3 install pytest requests pyOpenSSL
-cd build && make %{?_smp_mflags} check
+pip3 install pytest pyOpenSSL flake8
```

Comment:
Two things:
- Since we're here, could you freeze the versions of the external Python modules to keep the tests stable?
- We do have `python-flake8` as a package. Can we use that instead of the external one?

Reply:
I went ahead and added version specifications here for the packages specified. We offer flake8 in extended and I would hesitate to migrate it forward into core (and upgrade it) for this PR as the purpose of a minor upgrade. Alternatively I have locked the version inline on the install command.

Reply:
That was dumb of me - I missed the fact that `flake8` is an extended package. Sorry for the trouble.